webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -50,6 +60,21 @@ def test_dedent_left(self):
         assert dedent_left(text, 8) == 'Line 1\nLine 2\nLine 3'
 
 
+class TestIsValidModelClass():
+    """Test that is_valid_model_class validates API models."""
+
+    @pytest.mark.parametrize("model", (SuperSearch, SuperSearchUnredacted))
+    def test_valid(self, model):
+        assert is_valid_model_class(model)
+
+    @pytest.mark.parametrize(
+        "not_model",
+        ("SuperSearch", int, contextlib, SocorroMiddleware, ESSocorroMiddleware)
```

Comment:
Non-valid arguments:

* ``"SuperSearch"`` - a string, not a class
* ``int`` - a class, but not a subclass of ``SocorroMiddleware``
* ``contextlib`` - a module, not a class. This is how the bug would manifest in production.
* ``SocorroMiddleware`` - A disallowed subclass of ``SocorroMiddleware``
* ``ESSocorroMiddleware`` - A disallowed subclass of ``SocorroMiddleware``